sbi/inference/snle/snle_base.py

```diff
@@ -366,4 +367,10 @@ def _loss(self, theta: Tensor, x: Tensor) -> Tensor:
         Returns:
             Negative log prob.
         """
+        theta = reshape_to_batch_event(
+            theta, event_shape=theta.shape[1:]
+        )
+        x = reshape_to_iid_batch_event(
```

Comment:
sorry, I am again confused by the iid dims. Why is it set to `False` here? In SNLE, `x` could have a leading iid dim, no?

Reply:
See above, but let's also chat about this tomorrow.